Multisig/UI/UI Library/RefreshableScrollView.swift

```diff
@@ -87,16 +87,20 @@ struct RefreshableScrollView<Content: View>: View {
     }
 
     func symbolRotation(_ scrollOffset: CGFloat) -> Angle {
-        // We will begin rotation, only after we have passed
-        // 60% of the way of reaching the threshold.
         if scrollOffset < threshold * CGFloat(thresholdRubicon) {
             return .degrees(0)
         } else {
             // Calculate rotation, based on the amount of scroll offset
-            let h = Double(threshold)
-            let d = Double(scrollOffset)
-            let v = max(min(d - (h * thresholdRubicon), h * (1 - thresholdRubicon)), 0)
-            return .degrees(180 * v / (h * (1 - thresholdRubicon)))
+            let t = Double(threshold)
+            let tr = t * thresholdRubicon
+            let tri = t * (1 - thresholdRubicon)
```

Comment:
`t`, `tr`, `tri` didn't make it better either 😜 

How about:

```swift
// rename the `threshold` variable to `refreshTresholdInPoints` and `thresholdRubicon` in `relativeRotationThreshold`;

let rotationStartInPoints = Double(refreshTresholdInPoints) * relativeRotationThreshold
let rotationEndInPoints = Double(refreshTresholdInPoints) * (1 - relativeRotationThreshold)

if scrollOffset < rotationStartInPoints {
  return .degrees(0)
} else {
  let currentRotationOffset = max(min(Double(scrollOffset) - rotationStartInPoints, rotationEndInPoints), 0)
  let rotationAngle = Angle.degrees(180 * currentRotationOffset / rotationEndInPoints)
}
```

Reply:
ok. I changed.